Examples/example_pyb.py

```diff
@@ -10,19 +10,19 @@
 import time
 from rotary_irq_pyb import RotaryIRQ
 
-r = RotaryIRQ(pin_num_clk='X1', 
```

Comment:
Thanks for running the formatter against the repo files.  I should start doing that in my code.